docs/change-controlled-docs.rst

```diff
@@ -0,0 +1,293 @@
+         For example:
+
+         .. code-block:: bash
+
+            git cherry-pick <commit-sha>
+         
+         Also ensure that the document's change record is updated, but the ``draft`` option must remain in the document class command.
+
+      3. Push the backport branch to GitHub for continuous integration validation, rebase, and merge to master.
+         For example:
+
+         .. code-block:: bash
+
+            git checkout master
+            git pull
+            git checkout u/<username>/v<major>.<minor>-backport
+            git rebase -i master
+            git push -u  # --force
+            git checkout master
+            git merge --no-ff u/<username>/v<major>.<minor>-backport
+            git push
+
+.. _ccd-hotfix:
+
+Hotfixing a released document
+=============================
+
+The procedure above (:ref:`ccd-release`) describes how to make a new version of a document from the ``master`` branch.
+Sometimes it is necessary to hotfix a released document to fix a typo or make a similar minor change.
+In these cases you may not want to make a new release from the ``master`` branch because ``master`` has substantive, and unrelated, new content.
+Instead, you may hotfix a document from the release branch:
+
+1. Check out the head of the release branch for the version being fixed:
+
+   - For a project document:
+
+     .. code-block:: bash
+
+        git checkout tickets/LCR-<prev>
+
+   - For a DM document:
+
+     .. code-block:: bash
+
+        git checkout tickets/RFC-<prev>
+
+   ``<prev>`` is the RFC or LCR number of the document release being fixed.
+
+2. Create a ticket branch (the JIRA ticket is scoped for implementing the fix and coordinating the release):
+
+   .. code-block:: bash
+
+      git checkout -b tickets/DM-<N>
+      git push -u
+
+3. Commit fixes onto that ``tickets/DM-<N>`` branch and push to GitHub.
+
+4. Follow the steps in :ref:`ccd-release`, noting that the base branch is now ``tickets/DM-<N>``, not ``master``.
+   In the last step, the amendment commits (such as those on the ``tickets/DM-<N>`` branch and on the release branch) are still backported to ``master``.
+   The hotfix release branch is not merged onto the previous release branch.
+
+.. _ccd-docushare-upload:
+
+Uploading to DocuShare
+======================
+
+Follow these steps to upload a draft or released document to DocuShare:
+
+1. Send the PDF of the document to a person able to upload to DocuShare:
+
+   - For project documents (such as LPM and LSE), email the PDF to the LSST librarian.
+
+   - For DM documents (such as LDM and DMTR), email the PDF to a DM documentalist.
+
+2. Wait for the documentalist or librarian to upload the document and verify that it appears on the Version page of the document on DocuShare.
+   You can find the document version page with the short link ``https://ls.st/<handle>*``.
+   For example, `https://ls.st/ldm-151* <https://ls.st/ldm-151*>`_.
+
+3. Tag the commit that produced the DocuShare upload.
```

Comment:
This is duplicating information from earlier in the document. Do we need it again?

Reply:
Could you point to where this is duplicating information? I've arranged it so that whenever a DocuShare upload is called for, this section is referenced. I don't think I describe how to make a DocuShare tag anywhere else.

You might be thinking of the semantic tag, but that's distinct and occurs in only one place.

Let me know.

Reply:
Oops. Yes, I didn't spot the difference as the text was almost identical and I felt like I had a glitch in the Matrix.